### ISSUE TITLE
Questions about training ACGAN

### ISSUE THREAD

**Issue by zzw1123** [znxlwm/pytorch-generative-model-collections#14]
Thanks for your contribution!
A problem occurs when I am training ACGAN without tuning any parameters which is shown as follows:
As the training process goes on, the loss of D is declining, but the loss of G is increasing on the contrary...and the gap between two of them is pretty large. I do not know how to solve it...
![image](https://user-images.githubusercontent.com/16879220/42883369-09a29d50-8ace-11e8-8e87-4d128271f45f.png)

**Comment by furkanulger** [comment]
I see the same behaviour in training. Is there any update on this issue?
